blog/2015/03/first-software-carpentry-korea.html

```diff
@@ -2,6 +2,28 @@
 layout: blog
 root: ../../..
 author: Victor (Kwnagchun) Lee
+<<<<<<< HEAD
```

Comment:
@statkclee You need to fix this conflicts. Can you let me know when you did it so I can review it? If you need assistance to solve the conflicts, let us know.